waspc/cli/Command/Build.hs

```diff
@@ -14,14 +14,28 @@ import CompileOptions (CompileOptions (..))
 import Control.Monad.Except (throwError)
 import Control.Monad.IO.Class (liftIO)
 import qualified Lib
-import StrongPath (Abs, Dir, Path, (</>))
+import StrongPath (Abs, Dir, Path, toFilePath, (</>))
+import System.Directory
+  ( doesDirectoryExist,
+    removeDirectoryRecursive,
+  )
 
 build :: Command ()
 build = do
   waspProjectDir <- findWaspProjectRootDirFromCwd
   let outDir =
         waspProjectDir </> Common.dotWaspDirInWaspProjectDir
           </> Common.buildDirInDotWaspDir
+      buildDirFilePath = toFilePath outDir
+
+  liftIO $ putStrLn "Clearing the content of the .wasp/build directory..."
+  doesBuildDirExist <- liftIO $ doesDirectoryExist buildDirFilePath
+  if doesBuildDirExist
+    then liftIO $ do
+      removeDirectoryRecursive buildDirFilePath
```

Comment:
I would move the "Clearing the content... " inside the if-then, and would remove "Nothing to clean ..." completely. So if there is no build dir, there is no messaging, and if there is one, we first say that we are clearing the content and then we say we sucessfully cleared it.